sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -208,14 +208,16 @@ public partial class RequestConditions : Azure.MatchConditions
         public System.DateTimeOffset? IfModifiedSince { get { throw null; } set { } }
         public System.DateTimeOffset? IfUnmodifiedSince { get { throw null; } set { } }
     }
-    public partial class RequestContext : System.ServiceModel.Rest.InvocationOptions
+    public partial class RequestContext : System.ServiceModel.Rest.RequestOptions
     {
         public RequestContext() { }
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public Azure.ErrorOptions ErrorOptions { get { throw null; } set { } }
         public void AddClassifier(Azure.Core.ResponseClassificationHandler classifier) { }
         public void AddClassifier(int statusCode, bool isError) { }
         public void AddPolicy(Azure.Core.Pipeline.HttpPipelinePolicy policy, Azure.Core.HttpPipelinePosition position) { }
+        public void Apply(Azure.Core.HttpMessage message) { }
+        public override void Apply(System.ServiceModel.Rest.Core.PipelineMessage message) { }
```

Comment:
I don't like needing two of these on options.

Reply:
This might imply that options becomes templated as well to say "I can copy myself to this message type"?

We'll revisit this when we loop back to solve the construction patterns for pipeline/message, everything.